iwyu_output.h

```diff
@@ -321,6 +347,9 @@ class IwyuFileInfo {
   // Holds all the lines (#include and fwd-declare) that are reported.
   vector<OneIncludeOrForwardDeclareLine> lines_;
 
+  // Holds all of the using delcs that ar reported.
```

Comment:
"delcs -> decls and ar -> are"

Thanks.  In general it looks like my spelling is atrocious.

"'of' seems redundant here."

Not where I'm from. Potato, potato?